bitcoin/src/blockdata/script/witness_program.rs

```diff
@@ -17,6 +18,9 @@ use crate::blockdata::script::{PushBytes, PushBytesBuf, PushBytesErrorReport, Sc
 use crate::crypto::key::{CompressedPublicKey, TapTweak, TweakedPublicKey, UntweakedPublicKey};
 use crate::taproot::TapNodeHash;
 
+/// The maximum byte size of a segregated witness program.
+pub const WITNESS_PROGRAM_MAX_SIZE: usize = 40;
```

Comment:
We could just use `MAX_SIZE` with no loss of clarity, outside of this module folk can use `witness_program::MAX_SIZE`.

Reply:
We could have `MIN_SIZE` also.

Reply:
Good call, fixed